OpenXMLSDK.Engine/ReportEngine/DataContext/SubstitutableStringModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace OpenXMLSDK.Engine.ReportEngine.DataContext
+{
+    /// <summary>
+    /// Model class for a substitutable string value in context
+    /// </summary>
+    public class SubstitutableStringModel : BaseModel
+    {
+        /// <summary>
+        /// List of substitution texts
+        /// </summary>
+        public ContextModel DataSource { get; set; }
+
+        /// <summary>
+        /// Used to define the final rendering string You can set precision of other :
+        /// Ex : '{0:G2} kV' for the value 3.230 will generate '3.23 kV' string
```

Comment:
Could you update this exemple to use the <example> tag ?
doc : https://docs.microsoft.com/fr-fr/dotnet/csharp/programming-guide/xmldoc/example